fiftyone/utils/cvat.py

```diff
@@ -2377,6 +2425,2066 @@ def write(
+
+        Args:
+            task_id: the id of the task to which to upload data
+            paths: a list of paths to media files on disk to upload
+            image_quality (75): an int in `[0, 100]` determining the image
+                quality to upload to CVAT
+            job_assignees (None): a list of usernames to assign jobs
+            job_reviewers (None): a list of usernames to assign job reviews
+
+        Returns:
+            a list of the job ids created for the task
+        """
+        data = {"image_quality": image_quality}
+        files = {
+            "client_files[%d]" % i: open(p, "rb") for i, p in enumerate(paths)
+        }
+        self.post(self.task_data_url(task_id), data=data, files=files)
+
+        job_ids = []
+        while job_ids == []:
+            job_resp = self.get(self.jobs_url(task_id))
+            job_ids = [j["id"] for j in job_resp.json()]
+
+        if job_assignees is not None:
+            for ind, job_id in enumerate(job_ids):
+                assignee_ind = min(ind, len(job_assignees) - 1)
+                assignee = job_assignees[assignee_ind]
+                user_id = self.get_user_id(assignee)
+                if assignee is not None and user_id is not None:
+                    job_patch = {"assignee_id": user_id}
+                    self.patch(self.taskless_job_url(job_id), json=job_patch)
+
+        if job_reviewers is not None:
+            for ind, job_id in enumerate(job_ids):
+                reviewer_ind = min(ind, len(job_reviewers) - 1)
+                reviewer = job_reviewers[reviewer_ind]
+                user_id = self.get_user_id(reviewer)
+                if reviewer is not None and user_id is not None:
+                    job_patch = {"reviewer_id": user_id}
+                    self.patch(self.taskless_job_url(job_id), json=job_patch)
+
+        return job_ids
+
+    def upload_samples(
+        self,
+        samples,
+        label_schema,
+        media_field="filepath",
+        segment_size=None,
+        image_quality=75,
+        task_assignee=None,
+        job_assignees=None,
+        job_reviewers=None,
+    ):
+        """Parse the given samples and use the label schema to create tasks,
+        upload data, and upload formatted annotations to CVAT.
+
+        Args:
+            samples: a :class:`fiftyone.core.collections.SampleCollection` to
+                upload to CVAT
+            label_schema: a dictionary containing the description of label
+                fields, classes and attribute to annotate
+            media_field ("filepath"): string field name containing the paths to
+                media files on disk to upload
+            segment_size (None): maximum number of images to load into a job.
+                Not applicable to videos
+            image_quality (75): an int in `[0, 100]` determining the image
+                quality to upload to CVAT
+            task_assignee (None): the username to assign the created task(s)
```

Comment:
FYI I removed the word "sequentially" from these docstrings so that there could be one source of truth for documenting details like how assignees are chosen- the integration page docs